pwndbg/commands/version.py

```diff
@@ -189,7 +196,17 @@ def bugreport(run_browser=False):
     github_issue_url = "https://github.com/pwndbg/pwndbg/issues/new"
     github_issue_body = "?body=" + quote(issue_bugreport)
 
-    if run_browser:
+    if use_gh:
+        try:
+            with NamedTemporaryFile("w", delete=True) as f:
+                f.write(issue_bugreport)
+                f.flush()
+                check_call([os.environ.get("EDITOR", "vi"), f.name])
+                check_call(["gh", "issue", "create", "--body-file", f.name])
+        except Exception:
```

Comment:
We want to raise the original exception, so `except Exception as e:` and then `raise e`.